x-pack/plugins/ml/public/application/components/anomalies_table/anomaly_details_utils.tsx

```diff
@@ -276,5 +315,342 @@ export function getDetailsItems(anomaly: AnomaliesTableRecord, filter: EntityCel
     });
   }
 
-  return items;
+  return (
+    <>
+      {items.map(({ title, description }) => (
+        <>
+          <EuiFlexGroup gutterSize="none">
+            <EuiFlexItem css={{ width: '180px' }} grow={false}>
+              {title}
+            </EuiFlexItem>
+            <EuiFlexItem>{description}</EuiFlexItem>
+          </EuiFlexGroup>
+          <EuiSpacer size="xs" />
+        </>
+      ))}
+    </>
+  );
+};
+
+export const AnomalyExplanationDetails: FC<{ anomaly: AnomaliesTableRecord }> = ({ anomaly }) => {
+  const explanation = anomaly.source.anomaly_score_explanation;
+  if (explanation === undefined) {
+    return null;
+  }
+  const initialScore = Math.floor(1000 * anomaly.source.initial_record_score) / 1000;
+  const finalScore = Math.floor(1000 * anomaly.source.record_score) / 1000;
+  const scoreDifference = initialScore - finalScore;
+  const ACCEPTABLE_NORMALIZATION = 10;
+
+  const yes = i18n.translate('xpack.ml.anomaliesTable.anomalyExplanationDetails.yes', {
+    defaultMessage: 'Yes',
+  });
+  const no = i18n.translate('xpack.ml.anomaliesTable.anomalyExplanationDetails.no', {
+    defaultMessage: 'No',
+  });
+
+  const explanationDetails = [];
+  const anomalyType = getAnomalyType(explanation);
+  if (anomalyType !== null) {
+    explanationDetails.push({
+      title: (
+        <FormattedMessage
+          id="xpack.ml.anomaliesTable.anomalyDetails.anomalyExplanationDetails.anomalyType"
+          defaultMessage="Anomaly type"
+        />
+      ),
+      description: <>{anomalyType}</>,
+    });
+  }
+
+  if (scoreDifference > ACCEPTABLE_NORMALIZATION) {
+    explanationDetails.push({
+      title: (
+        <EuiToolTip
+          position="left"
+          content={i18n.translate(
+            'xpack.ml.anomaliesTable.anomalyDetails.anomalyExplanationDetails.recordScoreTooltip',
+            {
+              defaultMessage: 'Record score normalized blah blah blah',
```

Comment:
Description text needed

Reply:
@szabosteve and @valeriy42 can you please help me with this tooltip text.

Reply:
We have a lot of questions / SDHs where the final score is significantly different than the initial score. As well as the tooltip text here, I wonder if we need extra text (not behind a tooltip), possibly in the 'Impact on initial score' section clarifying why this is, giving reasons for 'why the actual score is so much lower than the initial score?'.

<img width="286" alt="image" src="https://user-images.githubusercontent.com/7405507/195556079-47ea9410-cc4e-4393-91c4-3f45dcfbb9b6.png">

Reply:
It looks like this is just a repeat of the "Initial record score" and the "Record score" from the left column, just with an arrow in between?  If that's true, having "normalized" in the label here seems to falsely imply the other values aren't normalized. Unless there's something else going on here that you want to cover in the tooltip?

Reply:
I was thinking that the label could be "Record score reduction" or something like that. With the tooltip explaining that normalization has caused the score to be reduced.

> I wonder if we need extra text (not behind a tooltip), possibly in the 'Impact on initial score' section clarifying why this is, giving reasons for 'why the actual score is so much lower than the initial score?'.

Yes I think that would be helpful.